WIP-enrollment-CIP.md

```diff
@@ -0,0 +1,17 @@
+In my current estimation, compared to the Plutus- and Ledger-enrolling CIPs above, any CIP enrolling Consensus would be trivial.
+We've simply had many less contributors and many fewer design iterations, each of which has been a massive investment.
+My relevant suggestions are essentially just a few sentences.
+
+> The authors of any CIP that explicitly affects the Cardano consensus protocol will need to communicate early and often with some combination of the IOG Researchers and the available experts on the various existing implementations.
+> We suggest beginning by scheduling a discussion on the agenda of a session of INTERSECT's Consensus Technical Working Group.
+> The discussion there will likely involve scheduling additional discussions with the aforementioned Researchers and experts (eg the IOE Consensus Team holds weekly office hours for this sort of communication; Amaru does not hold Office Hours yet, but is planning to).
```

Comment:
This CIP could be the seed from which to grow a proper process that would outlive the people actually implementing consensus now, which requires defining rules and process to change the rules. 
What kind of discussion? Over what kind of artefacts? Who would be part of these discussions? What would be the requirements for a CIP on Consensus to even be discussed by experts? What kind of SLA would experts agree on? etc.

Reply:
To draw the analogy with research: you want to do some research, you publish a paper in some venue, the reviewers review your paper, possibly requesting changes and explanations, you update your paper, you get final approval or rejection. 
There's a social contract and a clear process with some relatively well-known rules and SLAs (eg. you have to review papers throughly, and with deadlines).
In our case, a CIP could require some supporting code, perhaps a formal spec with some proofs about the implementation, conformance with previously known or provided test vectors, etc.